creativecloud/scripts/utils.js

```diff
@@ -86,9 +98,21 @@ function getDecorateAreaFn() {
         import(`${getConfig().codeRoot}/deps/interactive-marquee-changebg/changeBgMarquee.js`);
         break;
       }
-      case firstBlock?.classList.contains('marquee'):
-        firstBlock.querySelectorAll('img').forEach(eagerLoad);
+      case firstBlock?.classList.contains('marquee'): {
+        // Load image eagerly for specific breakpoint
+        const viewport = defineDeviceByScreenSize();
+        const lcpImageByViewport = {
+          MOBILE: 'div img',
+          TABLET: 'div:nth-child(2) img',
+          DESKTOP: 'div:nth-child(3) img',
+        };
+        const backgroundImages = firstBlock.querySelector('div');
+        if (backgroundImages.querySelectorAll('img').length === 1) eagerLoad(firstBlock.querySelector('div img'));
+        else eagerLoad(firstBlock.querySelector(lcpImageByViewport[viewport]));
```

Comment:
@Ruchika4 What about the case where there are only two images in background. This would not eagerload for desktop ad there would not be an nth-child(3)